nodejs/week1/avg.js

```diff
@@ -6,13 +6,10 @@ if (args.length == 0) {
 }
 
 const numbers = args.map(Number);
-
 if (numbers.some(isNaN)) {
   console.log("All arguments must be valid numbers.");
   process.exit(1);
 }
-
 const sum = numbers.reduce((acc, cur) => acc + cur, 0);
 const avg = sum / numbers.length;
-
-console.log(avg);
\ No newline at end of file
+console.log(avg);
```

Comment:
please try to give some explicit message like " average of ....." so code would be more readable .